roles/openshift_master/templates/v1_partials/oauthConfig.j2

```diff
@@ -77,6 +77,8 @@ oauthConfig:
   - name: {{ identity_provider.name }}
     challenge: {{ identity_provider.challenge }}
     login: {{ identity_provider.login }}
+{% set default_mapping_method = 'add' if openshift.master.identity_providers | length > 1 else 'claim' %}
+    mappingMethod: {{ identity_provider.mappingMethod | default(default_mapping_method) }}
```

Comment:
hmm... not sure... how explicitly should they have to indicate the different identity providers are addressing the same users? if they start with `claim`, and run into conflicts, switching to `add` makes everything better. if they start with `add` and accidentally join identities which really shouldn't be mapped to the same user, switching to `claim` makes new users work as expected, but it's harder to untangle the already-mapped identities and users